extensions/httpserver/httpserver_hsminweb.lua

```diff
@@ -1607,13 +1607,13 @@ module.formattedDate    = HTTPformattedDate
 
 --- hs.httpserver.hsminweb.urlParts(url) -> table
 --- Function
---- Parse the specified URL into it's constituant parts.
+--- Parse the specified URL into its constituent parts.
```

Comment:
I'm not generally looking at `it's`/`its` -- it's too expensive for a large repository. This was an `in-the-neighborhood` fix.